src/Components/PlaceCard/PlaceCard.js

```diff
@@ -5,12 +5,8 @@ import "./PlaceCard.scss";
 const PlaceCard = ({ places, city }) => {
   const eachPlace = places.map((place) => {
     return (
-      <Link
-        to={`/${city}/${place.id}`}
-        key={place.name}
-        className="place-thumb"
-      >
-        <div id={`${place.id}`} className="card-container">
+      <Link to={`/${city}/${place.id}`} className="place-thumb" key={place.id}>
```

Comment:
Does `city` mean anything in this context or is it just syntactical sugar for the user to see in the URL?

I imagine the `place.id` should give you the details to lookup everything you need when you get to that page.

Reply:
Do you recommend removing it?